cri/v1alpha2/cri.go

```diff
@@ -1359,6 +1361,9 @@ func (c *CriManager) PullImage(ctx context.Context, r *runtime.PullImageRequest)
 	}
 
 	if err := c.ImageMgr.PullImage(ctx, imageRef, authConfig, bytes.NewBuffer([]byte{})); err != nil {
+		if err == errtypes.ErrNotfound {
```

Comment:
unnecessary in CRI part. 1. useless HTTP type error here. 2. kubelet have not recognize the image `not found` error